applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.hpp

```diff
@@ -81,6 +83,79 @@ class UpdatedLagrangian
     KRATOS_DEFINE_LOCAL_FLAG( COMPUTE_RHS_VECTOR_WITH_COMPONENTS );
     KRATOS_DEFINE_LOCAL_FLAG( COMPUTE_LHS_MATRIX_WITH_COMPONENTS );
 
+    struct MaterialPointVariables
```

Comment:
I just want to confirm: are these variables created because the original variable registered under SetValue and GetValue are removed?

Reply:
Yes that's right, they are created on the local element now instead of the geometry due to changes in GeometryDataContainer - this is the core of this PR.